doc/02-installation.md

```diff
@@ -351,6 +351,11 @@ Please make sure to enable this repository beforehand.
 ```bash
 zypper install monitoring-plugins
 ```
+If you want to get more plugins, try to install the `monitoring-plugins-all` package instead. This package recommends all official plugins and additional packages that are available in the [server:monitoring repository](https://build.opensuse.org/project/repositories/server:monitoring):
+```bash
+zypper install monitoring-plugins-all
+```
+
```

Comment:
Actually it's "at all", not "more". Have a look at `rpm -q --filesbypkg monitoring-plugins`, that's basically nothing. In contrast `rpm -q --filesbypkg --requires monitoring-plugins-all` shows.. even less?! @htriem Can you confirm this?

Reply:
A better query might be: `rpm -q --recommends monitoring-plugins-all`

On my system, this gives me the following list of packages, that would be installed:
```bash
monitoring-plugins-apt
monitoring-plugins-bgpstate
monitoring-plugins-bind
monitoring-plugins-bonding
monitoring-plugins-breeze
monitoring-plugins-by_ssh
monitoring-plugins-clamav
monitoring-plugins-cluster
monitoring-plugins-contentage
monitoring-plugins-cups
monitoring-plugins-dbi-mysql
monitoring-plugins-dbi-pgsql
monitoring-plugins-dbi-sqlite3
monitoring-plugins-dhcp
monitoring-plugins-dig
monitoring-plugins-disk
monitoring-plugins-disk_smb
monitoring-plugins-diskio
monitoring-plugins-dns
monitoring-plugins-dummy
monitoring-plugins-file_age
monitoring-plugins-flexlm
monitoring-plugins-fping
monitoring-plugins-game
monitoring-plugins-hpasm
monitoring-plugins-hpjd
monitoring-plugins-http
monitoring-plugins-icmp
monitoring-plugins-ide_smart
monitoring-plugins-ifoperstatus
monitoring-plugins-ifstatus
monitoring-plugins-ipmi-sensor1
monitoring-plugins-ircd
monitoring-plugins-ldap
monitoring-plugins-linux_raid
monitoring-plugins-load
monitoring-plugins-log
monitoring-plugins-mailq
monitoring-plugins-maintenance
monitoring-plugins-mem
monitoring-plugins-mrtg
monitoring-plugins-mrtgtraf
monitoring-plugins-mysql
monitoring-plugins-mysql_health
monitoring-plugins-nagios
monitoring-plugins-nfsmounts
monitoring-plugins-nis
monitoring-plugins-nt
monitoring-plugins-ntp_peer
monitoring-plugins-ntp_time
monitoring-plugins-nwstat
monitoring-plugins-oracle
monitoring-plugins-overcr
monitoring-plugins-pgsql
monitoring-plugins-ping
monitoring-plugins-procs
monitoring-plugins-qlogic_sanbox
monitoring-plugins-radius
monitoring-plugins-real
monitoring-plugins-rpc
monitoring-plugins-rsync
monitoring-plugins-sensors
monitoring-plugins-smtp
monitoring-plugins-snmp
monitoring-plugins-ssh
monitoring-plugins-swap
monitoring-plugins-tcp
monitoring-plugins-time
monitoring-plugins-ups
monitoring-plugins-ups_alarm
monitoring-plugins-uptime
monitoring-plugins-users
monitoring-plugins-wave
monitoring-plugins-zypper
nagios-xen-host
```